src/pages/growers/[growerid].js

```diff
@@ -435,7 +385,7 @@ export default function Planter(props) {
               mt: [5, 10],
             }}
           >
-            {planter.species.species.map((species) => (
+            {grower.species.species.map((species) => (
               <TreeSpeciesCard
```

Comment:
At the moment, each entity (are these referred to as `species`, `tree`s, `capture`s, or something else?) in the `grower.species.species` array are missing `desc` and `total` properties and look like this:

```
{
  age: null,
  attributes: null,
  created_at: "2022-01-25T12:06:36.000Z",
  estimated_geographic_location: "0101000020E61000005E0A2435A8682AC0CDBB02E82FDC2040",
  estimated_geometric_location: "0101000020E61000005E0A2435A8682AC0CDBB02E82FDC2040",
  gps_accuracy: 0,
  grower_id: 940,
  id: "da4d3ed8-8655-44c5-a7ba-d4c45a0dfb10",
  image_url: "http://localhost:3000/7230cc00-f2ab-4786-8df8-dbc58bad5c0c",
  lat: "8.430052996",
  latest_capture_id: "b74f65d8-bd25-4ca4-a4f7-8ad50ba9a936",
  lon: "-13.204408322",
  morphology: null,
  name: "The Best Tree",
  species_id: null,
  status: "active",
  updated_at: "2022-01-25T12:06:36.000Z"
}
```

If this will be addressed in another `v2` issue, I can leave it for now.